examples/snapshot/test/specs/test.e2e.ts

```diff
@@ -0,0 +1,15 @@
+import { expect } from '@wdio/globals'
+
+describe('My Login application', () => {
+    it('should login with valid credentials', async () => {
+        await browser.url(`https://the-internet.herokuapp.com/login`)
+
+        await $('#username').setValue('tomsmith')
+        await $('#password').setValue('SuperSecretPassword!')
+        await $('button[type="submit"]').click()
+
+        await expect($('#flash')).toMatchSnapshot()
+        await expect({a:"a"}).toMatchSnapshot()
+        await expect({a2:"a"}).toMatchSnapshot()
```

Comment:
This makes me think: having support for `toMatchSnapshot` is good for running unit/component testing in the browser. I suggest to add an matcher called `toMatchElementSnapshot` which compares the html structure of an element.